### PR TITLE
Test against RGeo3

### PR DESCRIPTION
**Pull request by keithdoggett**
I had to change a few test cases to work with the GEOS WKT generator. The library shouldn't have needed any changes since this doesn't directly depend on `rgeo` and only depends on `rgeo-activerecord`, but it's probably a good idea to test anyways... and it doesn't need any changes.

I added `rgeo` to the gemfile since bundler won't install prerelase versions without explicitly setting them somewhere, but I will remove that once version 3 is released.

### CODE REVIEW

File: test/basic_test.rb [hunk 1]
```diff
@@ -196,11 +196,11 @@ def test_multi_polygon_column
     end
     SpatialModel.reset_column_information
     rec = SpatialModel.new
-    wkt = "MULTIPOLYGON (((-73.97210545302842 40.782991711401195, " \
+    wkt = "MULTIPOLYGON (((-73.97210545302842 40.78299171140119, " \
```
Comment (BuonOmo) [inline test/basic_test.rb:199]: Why this change ?

Comment (keithdoggett) [reply test/basic_test.rb:199]: I'm not sure. For whatever reason the 15th decimal in this wkt was getting truncated on the output. I tried looking into it a bit, but I'm not sure if it's a PostGIS change or something to do with the WKT parsers in RGeo. I'll see if I can figure out exactly why it's failing.

Also kind of related but this PR (https://github.com/rgeo/activerecord-postgis-adapter/pull/373) pointed out that we use the Ruby based parsers here. I can open a separate PR that uses the Geos based parsers if available.

Comment (keithdoggett) [reply test/basic_test.rb]: @BuonOmo it works now so i'm not sure what was going on before.
